tools/netsh/ebpfnetsh.rc

```diff
@@ -67,14 +68,17 @@ BEGIN
 \n      type      - A string used as if it were a section name, to\
 \n                  determine the program type and expected attach\
 \n                  type.  If not specified, the default is xdp.\
-\n      pinned    - Optionally the name to pin the program to.  If not\
-\n                  specified, the default is to not pin the program.\
+\n      pinpath   - Optionally the name to pin the program to.\
+\n      pinned    - One of the following values:\
+\n                   none: Load all programs without pinning any.\
+\n                   first: Pin the first program in the file.\
+\n                   all: Pin all programs in the file.\
 \n      execution - One of the following values:\
 \n                   jit: Convert the program to machine code.\
 \n                   interpret: Run the program as byte code.\
 \n\
-\nRemarks: Loads a program (if not already loaded) and pins it if\
-\n         requested.\
+\nRemarks: Loads programs from a file (if not already loaded)\
```

Comment:
How do we check if the program is not already loaded?

Reply:
you can do "show programs" beforehand

Reply:
Removed parenthetical phrase